arch/inst/Zawrs/wrs.nto.yaml

```diff
@@ -10,18 +10,22 @@ description: |
   includes all the bytes of the memory location using the LR instruction. Then a subsequent `wrs.nto`
   instruction would cause the hart to temporarily stall execution in a low-power state until a store
   occurs to the reservation set or an interrupt is observed.
+
   This instruction is not supported in a constrained LR/SC loop.
   While stalled, an implementation is permitted to occasionally terminate the stall and complete
-  execution for any reason. `wrs.nto` follows the rules of the WFI instruction for resuming execution
+  execution for any reason.
+  `wrs.nto` follows the rules of the WFI instruction for resuming execution
   on a pending interrupt. When the TW (Timeout Wait) bit in `mstatus` is set and `wrs.nto` is executed
   in any privilege mode otherthan M mode, and it does not complete within an implementation-specific
   bounded time limit, the `wrs.nto` instruction will cause an illegal instruction exception.
   When executing in VS or VU mode, if the VTW bit is set in `hstatus`, the TW bit in `mstatus` is clear,
```

Comment:
add blank lines here, too:
- after "reason."
- after "pending interrupt."
- after "exception."
... all to match the paragraph structure in the spec.